eclipse-scout-core/src/form/fields/listbox/ListBox.less

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) 2010, 2025 BSI Business Systems Integration AG
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+.list-box.hide-table-lines {
```

Comment:
Maybe merge the three first lines to reduce the nesting level?

Reply:
changed accordingly